src/api/migraphx.py

```diff
@@ -450,4 +450,9 @@ def experimental_custom_op(h):
     h.virtual('compute_shape',
               api.params(inputs='std::vector<migraphx::shape>'),
               returns='migraphx::shape')
+    # TODO(umang) : If output aliasing then, runs_on_offload_flag must be false
```

Comment:
Is this still a TODO or has this been added?

Reply:
It is still a TODO